torch/fx/experimental/graph_gradual_typechecker.py

```diff
@@ -198,6 +200,53 @@ def bn2d_inference_rule(n: Node, op_type):
     else:
         raise TypeError(f'Cannot apply {op_type} with input type { arg_type} and existing type {n.type} on {n}')
 
+def calculate_hout(h_in, op_type):
+    if h_in == Dyn:
+        return Dyn
+
+    elif isinstance(h_in, int):
+        h_out = floor((h_in + (2 * op_type.padding[0] - op_type.dilation[0] *
+                               (op_type.kernel_size[0] - 1) - 1)) / op_type.stride[0]) + 1
+        return h_out
+    else:
+        raise TypeError(f'{h_in} must be a number or Dyn')
+
+def calculate_wout(w_in, op_type):
+    if w_in == Dyn:
+        return Dyn
+
+    elif isinstance(w_in, int):
+        w_out = floor((w_in + (2 * op_type.padding[1] - op_type.dilation[1] *
```

Comment:
nit: I would rewrite `floor(...)` to just use floor division `(op_type.kernel_size[1] - 1) - 1)) /` -> `(op_type.kernel_size[1] - 1) - 1)) //`

it's probably unlikely that floor(`x/y`) is different than `x // y` but sometimes floating point math is weird like that lol, and i think doing it this way is more in line with how it is computed in eager..